### ISSUE TITLE
prefix issue with step

### ISSUE THREAD

**Issue by danielehrhardt** [saadeghi/daisyui#371]
I am using a prefix for daisyui with the tailwind options. 
The problem is that the CSS Selector does not work correctly: 
Does not work:
```
<ul class="tw-steps">
<li class="tw-step">
```

Does work (but is wrong):
```
<ul class="tw-steps">
<li class="tw-step step">
```

![image](https://user-images.githubusercontent.com/19570043/147857270-b409a434-c38d-4aa2-a282-6e3f344903d4.png)

This Issue may occur also on other components.

**Comment by saadeghi** [comment]
@danielehrhardt yes. This issue applies for every daisyUI responsive component that contains other responsive utility classes.  
Unfortunately there are no other way to make components responsive other that using `[class~=classname]` selectors [[reason](https://github.com/tailwindlabs/tailwindcss/issues/3361#issuecomment-927879795)]  
And these selectors won't get prefix.  

Currently I can't do anything about it.

**Comment by saadeghi** [comment]
Fixed in v2.0  
https://play.tailwindcss.com/IFdiVRSNqD

**Comment by danielehrhardt** [comment]
> Fixed in v2.0  
> https://play.tailwindcss.com/IFdiVRSNqD

Amazing!!
